services/common/src/components/documents/ReplaceDocumentModal.tsx

```diff
@@ -153,4 +153,14 @@ const mapDispatchToProps = (dispatch) =>
     dispatch
   );
 
-export default connect(null, mapDispatchToProps)(ReplaceDocumentModal);
+export default compose(
```

Comment:
I guess this is more a preference thing for me as what you have looks like it should work, but we do have a `FormWrapper` component that handles this quite nicely without having to use the compose feature.

You can simply wrap the elements in it like this:

```
<FormWrapper
              name={name}
              onSubmit={() => {}}
              initialValues={initialValues}
              isEditMode={isEditMode}
              reduxFormConfig={
                reduxFormConfig ?? {
                  touchOnBlur: true,
                  touchOnChange: false,
                  enableReinitialize: true,
                }
              }
            >
            <div>...</div>
            </FormWrapper>
```

Reply:
Not a deal breaker, but I find it much easier to work with personally.  Especially by making the export simpler and cleaner for typescript types.